CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+## [1.1.0](https://github.com/dequelabs/axe-linter-action/compare/v1.0.0...v1.1.0) (2025-03-20)
+
+### Features
+
+- add filename and line number to error ([#60](https://github.com/dequelabs/axe-linter-action/issues/60)) ([943d86d](https://github.com/dequelabs/axe-linter-action/commit/943d86da9710318d858b418c86cd14c17bf9c0e5))
+
+### Bug Fixes
+
+- add debug flag when executing bash ([#35](https://github.com/dequelabs/axe-linter-action/issues/35)) ([467fcc5](https://github.com/dequelabs/axe-linter-action/commit/467fcc5c7b458fba3ad0dd81902719bad951bf8d))
+- Skipping Empty Files ([#51](https://github.com/dequelabs/axe-linter-action/issues/51)) ([bd47f92](https://github.com/dequelabs/axe-linter-action/commit/bd47f92b8ae7cfbc3ce9a8e8df0836db1876b974))
+
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## v1.0.0 (2025-03-20)
```

Comment:
This date should probably be the commit date of [the v1.0.0 commit](https://github.com/dequelabs/axe-linter-action/commit/9adc1d96f704d36cbe32e3be8af1373b37df7269)